coding/zip_creator.cpp

```diff
@@ -129,13 +74,25 @@ bool CreateZipFromFiles(std::vector<std::string> const & files, std::string cons
     return false;
 
   auto const compressionLevel = GetCompressionLevel(compression);
-  zip::FileInfo const fileInfo = {};
+  zip::FileInfo fileInfo = {};
+  FillZipLocalDateTime(fileInfo.tmz_date);
 
   try
   {
-    for (auto const & filePath : files)
+    int suffix = 0;
+    for (size_t i = 0; i < files.size(); i++)
     {
-      if (zip::Code::Ok != zip::OpenNewFileInZip(zip.Handle(), filePath, fileInfo, "",
+      auto const & filePath = files.at(i);
+      std::string fileInArchive = filesInArchive.at(i);
+      if (!strings::IsASCIIString(fileInArchive))
+      {
+        if (suffix == 0)
+          fileInArchive = "OrganicMaps.kml";
```

Comment:
1. Can the original extension be preserved?
2. Won't it replace the `folder/file.ext` with `file.ext`?

Reply:
> 1. Can the original extension be preserved?

We always export to kml, so looks like it is the only extension?

>     2. Won't it replace the `folder/file.ext` with `file.ext`?

Yes, I broke this thing and some ascii check, will fix it and cover with test.

Reply:
What if bookmarks folder contain images?

Reply:
Do we have a support for images inside kmz now? I was able to find m_imageUrl field, but it looks like an url to external website?